source/documentation/other-topics/access-cross-aws-resources.html.md.erb

```diff
@@ -0,0 +1,259 @@
+    type = map(string)
+    default = {
+      business-unit          = "Cloud Platform"
+      application            = "My Application"
+      is-production          = "false"
+      environment-name       = "Development"
+      owner                  = "cloud-platform"
+      infrastructure-support = "platforms@digital.justice.gov.uk"
+    }
+    }
+
+    resource "aws_iam_role" "my-namespace-app-iam-role" {
+    name               = "my-namespace-app-iam-role"
+    description        = "IAM role for My app to access s3 bucket - AAA in 11111 AWS account"
+    tags               = var.my-namespace-tags
+    assume_role_policy = data.aws_iam_policy_document.my-namespace-kiam-trust-chain.json
+    }
+
+    resource "kubernetes_secret" "iam-role-creds-my-app" {
+    metadata {
+      name      = "iam-role-creds-my-app"
+      namespace = "my-namespace"
+    }
+
+    data = {
+      arn       = aws_iam_role.my-namespace-app-iam-role.arn
+      name      = aws_iam_role.my-namespace-app-iam-role.name
+      unique_id = aws_iam_role.my-namespace-app-iam-role.unique_id
+    }
+    }
+
+    data "aws_iam_policy_document" "my-namespace-app-iam-role" {
+
+    # allow pods to assume this role
+    statement {
+      actions   = ["sts:AssumeRole"]
+      resources = [aws_iam_role.my-namespace-app-iam-role.arn]
+    }
+
+    # Provide list of permissions and target AWS account resources to allow access from
+    statement {
+      actions = [
+        "s3.PutObject",
+      ]
+      resources = [
+        "arn:aws:s3:::AAAAAAAA/AA-bucket/folder/*",
+      ]
+    }
+    }
+
+    resource "aws_iam_policy" "my-namespace-app-iam-policy" {
+    name   = "my-namespace-app-iam-policy"
+    policy = data.aws_iam_policy_document.my-namespace-app-iam-policy.json
+    }
+
+    resource "aws_iam_role_policy_attachment" "my-namespace-app-iam-policy" {
+    role       = aws_iam_role.my-namespace-app-iam-role.name
+    policy_arn = aws_iam_policy.my-namespace-app-iam-policy.arn
+    }
+    ```
+    </p>
+    </details>
+
+
+2. **Annotate the pod with the IAM role**
+
+    Once the PR in **step 1** is merged and applied, you will get the role ARN, name and id as a kubernetes secret in your namespace.
+
+    Decode the role ARN and annotate the pod to indicate which role which will need access to resource in different AWS account like below:
+
+    ```
+      annotations:
+        iam.amazonaws.com/role: <role-arn>
+    ```
+    An example would look like this:
+
+    ```
+      annotations:
+        iam.amazonaws.com/role: arn:aws:iam::11111111:role/my-namespace-app-iam-role
+    ```
+
+3. **Annotate the namespace to allow the role to AssumeRole**
+
+    Next, annotate the namespace to indicate which roles are permitted to be assumed within the namespace. 
+    
+    Update the `annotations` section with the below code along with existing annotations list in `00-namespace.yaml` 
+    for your namespace in the (cloud-platform-environments)[] repo and raise a PR.
+
+    ```
```

Comment:
Same here - add a bit of context to show exactly where in the file this snippet needs to go.